include/flatbuffers/flatbuffers.h

```diff
@@ -1557,6 +1557,7 @@ class Table {
 // C++ data structures).
 // Contains no functionality, purely documentative.
 struct NativeTable {
+  virtual ~NativeTable() {}
```

Comment:
Ok, I'll revert this change, though most of the modern (after 2010) compilers should know how to devirtualize (where possible) any functions
I choose this approach because it make the code simpler, smaller & nicer ...